lib/texture.js

```diff
@@ -828,6 +842,10 @@ module.exports = function createTextureSet (
       gl.copyTexImage2D(
         target, miplevel, format, info.xOffset, info.yOffset, width, height, 0)
     } else {
+      if (!data) {
```

Comment:
Firefox requires texture to be init with data before `subimage` call, otherwise gives warning.